src/components/Feedback/controls/LikeControl.tsx

```diff
@@ -22,42 +24,49 @@ type LikeControlProps = {
 const b = block('dc-feedback');
 
 const LikeControl = memo(
-    forwardRef<HTMLButtonElement, LikeControlProps>(({isLiked, view, onClick}, ref) => {
-        const {t} = useTranslation('feedback');
-        const {isVerticalView, popupPosition, controlClassName} = useContext(ControlsLayoutContext);
-        const tooltipText = isLiked ? t('cancel-like-text') : t('like-text');
+    forwardRef<HTMLButtonElement, LikeControlProps>(
+        ({isLiked, isPopupVisible, view, onClick}, ref) => {
+            const {t} = useTranslation('feedback');
+            const {isVerticalView, popupPosition, controlClassName} =
+                useContext(ControlsLayoutContext);
+            const tooltipText = isLiked ? t('cancel-like-text') : t('like-text');
```

Comment:
if the value is used only once, it is not necessary to put it in a separate variable

Reply:
Sorry, I don't understand what part of the code you're talking about, because what you highlighted I didn't write, it was written outside my pull-request